src/core/RdfGraph.ts

```diff
@@ -0,0 +1,51 @@
+import { Store as N3QuadStore } from 'n3';
+import { patchGraphState } from './patch';
+import { GraphPatch, GraphState, RdfPatch, UiSymbolProvider } from './types';
+
+export type RdfGraphOptions = {
+	symbolProvider: UiSymbolProvider;
+};
+
+export class RdfGraph {
+	#state: GraphState;
+	#quadStore: N3QuadStore;
+	#symbolProvider?: UiSymbolProvider;
+
+	constructor(options?: Partial<RdfGraphOptions>) {
+		this.#state = {
+			nodeStore: {},
+			predicateNodeStore: {},
+			edgeStore: {},
+		};
+		this.#quadStore = new N3QuadStore();
```

Comment:
Hva skjer med # foran navn? Er det en konvensjon?

Reply:
# er native JS for private field